src/pages/profile/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { NextPage } from "next";
+import Page from "src/components/Page";
+import ProfileContent from "src/pageContent/Profile";
+
+const Profile: NextPage = () => {
+  return (
+    <Page
+      bgImage="profile"
```

Comment:
Hem what does this line do? Right now putting `bgImage="profile"` won't give you any background image. What should you do in order to add a background image?

Reply:
Right now assets does not contain my desired background image in the folder. First I need to add a background image in "assets". Then I must go to tailwind.config.js to add the code 
profile: "url('/assets/backgrounds/profile.jpg')"